build.rb

```diff
@@ -68,7 +68,7 @@
   setup/gh_cli
   ubuntu_gcloud
   dotfiles
-  setup/windows_ssh
+  setup/ssh_agent
```

Comment:
The partial was renamed in the web setup because it's not specific to Windows but was included in the Linux setup too.